src/triggers.c

```diff
@@ -932,12 +932,24 @@ void hurt_on()
 	self->s.v.solid = SOLID_TRIGGER;
 	self->s.v.nextthink = -1;
 	setorigin(self, PASSVEC3(self->s.v.origin)); // it matter
+	g_globalvars.force_retouch = 2; // make sure even still objects get hit
```

Comment:
Why do we need this here? As far as I can see, 'force_retouch' is not really used...
There are 2 places where it is assigned 2:
- spawn_tdeath()
- teleport_use()
What is the purpose of this variable, if its value is never used?

Reply:
It was suggested by qqshka. I had another workaround earlier. The problem was that when flag carrier fell into the void and got gibbed, the rune fired the hurt event, but the flag did not. One guess is that runes have higher velocity which causes them to re-enter the trigger later than the flag but I'm not entirely sure. I first added a check if player died from T_Damage, and if so - didn't trigger the think and didn't set to SOLID_NOT which worked as well.

Reply:
wrt force_retouch, it's interacting with mvdsv,

https://github.com/QW-Group/mvdsv/blob/646e11e71b2f85e5d069f4437b5c37218526a0ae/src/sv_phys.c#L991-L992

Reply:
As @dsvensson mention `force_retouch` required in order `touch` function to be called since `solid` field manipulation of the `trigger_hurt`, 10 years ago I tried to fix same issue with line 934 where `setorigin()` executed but it helps for players entities only so we have to resort to `force_retouch`.

Reply:
okay guys, thanks for the detailed explanation!